tests/policy.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
+
+setup() {
+    load ../helpers/helpers.sh
+    wait_pods
+}
+
+teardown_file() {
+    load ../helpers/helpers.sh
+    kubectl delete pods --all
+    kubectl delete ns shouldbeignored --ignore-not-found
+}
+
+# Number of policies included in the recommended policies
+POLICY_NUMBER=6
+
+@test "[Policy tests] Install default policies in protect mode" {
```

Comment:
When you add more policy tests keep in mind that enforced recommended policies can affect them.